[miio] NPE after installing binding

I get the following NPE after installing the miio binding.

I'm on build 2077.

```
2020-12-16 10:19:44.119 [ERROR] [org.openhab.binding.miio            ] - bundle org.openhab.binding.miio:3.0.0.202012160829 (262)[org.openhab.binding.miio.internal.discovery.MiIoDiscovery(404)] : The activate method has thrown an exception
java.lang.NullPointerException: null
	at org.openhab.binding.miio.internal.discovery.MiIoDiscovery.getCloudDiscoveryMode(MiIoDiscovery.java:98) ~[?:?]
	at org.openhab.binding.miio.internal.discovery.MiIoDiscovery.startBackgroundDiscovery(MiIoDiscovery.java:119) ~[?:?]
	at org.openhab.core.config.discovery.AbstractDiscoveryService.activate(AbstractDiscoveryService.java:371) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at org.apache.felix.scr.impl.inject.methods.BaseMethod.invokeMethod(BaseMethod.java:228) ~[bundleFile:?]
	at org.apache.felix.scr.impl.inject.methods.BaseMethod.access$500(BaseMethod.java:41) ~[bundleFile:?]
	at org.apache.felix.scr.impl.inject.methods.BaseMethod$Resolved.invoke(BaseMethod.java:664) ~[bundleFile:?]
	at org.apache.felix.scr.impl.inject.methods.BaseMethod.invoke(BaseMethod.java:510) [bundleFile:?]
	at org.apache.felix.scr.impl.inject.methods.ActivateMethod.invoke(ActivateMethod.java:317) [bundleFile:?]
	at org.apache.felix.scr.impl.inject.methods.ActivateMethod.invoke(ActivateMethod.java:307) [bundleFile:?]
	at org.apache.felix.scr.impl.manager.SingleComponentManager.createImplementationObject(SingleComponentManager.java:340) [bundleFile:?]
	at org.apache.felix.scr.impl.manager.SingleComponentManager.createComponent(SingleComponentManager.java:114) [bundleFile:?]
	at org.apache.felix.scr.impl.manager.SingleComponentManager.getService(SingleComponentManager.java:982) [bundleFile:?]
	at org.apache.felix.scr.impl.manager.SingleComponentManager.getServiceInternal(SingleComponentManager.java:955) [bundleFile:?]
	at org.apache.felix.scr.impl.manager.SingleComponentManager.getService(SingleComponentManager.java:900) [bundleFile:?]
	at org.eclipse.osgi.internal.serviceregistry.ServiceFactoryUse$1.run(ServiceFactoryUse.java:212) [org.eclipse.osgi-3.12.100.jar:?]
	at java.security.AccessController.doPrivileged(Native Method) [?:?]
	at org.eclipse.osgi.internal.serviceregistry.ServiceFactoryUse.factoryGetService(ServiceFactoryUse.java:210) [org.eclipse.osgi-3.12.100.jar:?]
	at org.eclipse.osgi.internal.serviceregistry.ServiceFactoryUse.getService(ServiceFactoryUse.java:111) [org.eclipse.osgi-3.12.100.jar:?]
	at org.eclipse.osgi.internal.serviceregistry.ServiceConsumer$2.getService(ServiceConsumer.java:45) [org.eclipse.osgi-3.12.100.jar:?]
```

--- a/bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/discovery/MiIoDiscovery.java
+++ b/bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/discovery/MiIoDiscovery.java
@@ -95,11 +95,16 @@ public class MiIoDiscovery extends AbstractDiscoveryService {
         if (miioConfig != null) {
             try {
                 Dictionary<String, @Nullable Object> properties = miioConfig.getProperties();
-                String cloudDiscoveryModeConfig = (String) properties.get("cloudDiscoveryMode");
-                if (cloudDiscoveryModeConfig == null) {
+                String cloudDiscoveryModeConfig;
+                if (properties == null) {
                     cloudDiscoveryModeConfig = DISABLED;
                 } else {
-                    cloudDiscoveryModeConfig = cloudDiscoveryModeConfig.toLowerCase();
+                    cloudDiscoveryModeConfig = (String) properties.get("cloudDiscoveryMode");
+                    if (cloudDiscoveryModeConfig == null) {
+                        cloudDiscoveryModeConfig = DISABLED;
+                    } else {
+                        cloudDiscoveryModeConfig = cloudDiscoveryModeConfig.toLowerCase();
+                    }
                 }
                 return Set.of(SUPPORTED, ALL).contains(cloudDiscoveryModeConfig) ? cloudDiscoveryModeConfig : DISABLED;
             } catch (ClassCastException | SecurityException e) {

